docs/source/specification.md

```diff
@@ -83,7 +67,7 @@ There might be good reasons why any particular TRE does not possess one or more
 
 ### Specification Pillars and Capabilities
```

Comment:
```suggestion
## Specification Pillars and Capabilities
```

Looking at the draft, this might make more sense at the same header level as `Architectural Principles` rather than nested under it.